src/index.js

```diff
@@ -10,7 +10,7 @@ const input = document.getElementById('input');
 let stringData = JSON.stringify(todos.List);
 let listedItem;
 input.addEventListener('keydown', (evt) => {
```

Comment:
- [ ] [REQUIRED] You have done a great implementation, however, there is still something remaining you need to implement. You have ensured that the user is able to mark task completion by selecting the corresponding checkbox, however, you also need to ensure that the updated tasks list will be stored in local storage. It should be updated (`completed`: `true` / `false`) in the local storage. Currently, when a task is marked, the `Completed`: stays as `false` always, it does not get updated.

Reply:
Hiya Whoistolu, I understand. Actually, it is changing true/false but yeah It's not being reflected in localStorage, I'll see to that

Reply:
Alright, kindly fix it to get updated in the local storage.

Reply:
This change has been implemented well